k8s/demo/dbench/dbench-jdefault-r1.yaml

```diff
@@ -0,0 +1,55 @@
+kind: PersistentVolumeClaim
```

Comment:
I am kind of lost with these yamls?
Can we have folder structure that help reader understand better?
e.g.
`perf/0.8.0/dbench/hostpath/`
`perf/0.8.0/dbench/jiva-pv/`
`perf/0.8.0/dbench/cstor-gpd/`
`perf/0.8.0/dbench/cstor-local/`

Reply:
How are we going to showcase performance variation across openebs versions?